docker/php/Dockerfile

```diff
@@ -16,7 +20,7 @@ RUN { \
         echo 'opcache.memory_consumption=128'; \
         echo 'opcache.interned_strings_buffer=8'; \
         echo 'opcache.max_accelerated_files=4000'; \
-        echo 'opcache.revalidate_freq=0'; \
+        echo 'opcache.revalidate_freq=60'; \
```

Comment:
Can we keep zero? This parameter to 60 is great in production, but not in local. You would not see your changes immediately.